CMakeLists.txt

```diff
@@ -567,6 +567,49 @@ if(HPX_WITH_CUDA AND HPX_WITH_HIP)
   )
 endif()
 
+# ##
+# HPX STDEXEC configuration
+# ##
+
+hpx_option(
+  HPX_WITH_STDEXEC
+  BOOL
+  "Use STDEXEC executors instead of native HPX.(default: OFF)"
+  OFF
+  CATEGORY "Executor"
+  ADVANCED
+)
+
+hpx_option(
+  HPX_WITH_FETCH_STDEXEC
+  BOOL
+  "Use FetchContent to fetch STDEXEC.(default: OFF)"
+  OFF
+  CATEGORY "Executor"
+  ADVANCED
+)
+
+# if STDEXEC_ROOT was provided, HPX_WITH_STDEXEC is set to ON
+if(STDEXEC_ROOT OR HPX_WITH_FETCH_STDEXEC)
```

Comment:
This might have to be `Stdexec_ROOT` to avoid warnings with recent CMake versions

Reply:
Changed it, thank you